docs/add_sample.rst

```diff
@@ -44,6 +44,28 @@ the current one or be combined with it.
 .. note::
     Combining models is only possible for mesh samples not volumes.
 
+*********************
+View model properties
+*********************
+In order to view the properties of the sample click **View > Other Windows > Sample Properties**.
+
+For mesh samples- the memory (in mb), faces and vertices are displayed. 
+
+.. image:: images/mesh_sample_properties.png
+   :scale: 80
+   :alt: Mesh sample properties dialog 
+   :align: center
+
+For volume samples- the memory (in mb), dimensions, voxel size, minimum intensity and maximum intensity are displayed. 
+
+.. image:: images/volume_sample_properties.png
```

Comment:
Change the extension of the 2 images from .PNG to .png. On Linux, the two are not he same

Reply:
converted the image extension from PNG to png.